FieldOpt/Optimization/constraints/combined_spline_length_interwell_distance.cpp

```diff
@@ -13,7 +13,15 @@ namespace Optimization {
             dist_constr_settings.wells = settings.wells;
             dist_constr_settings.min = settings.min_distance;
             distance_constraint_ = new InterwellDistance(dist_constr_settings, variables);
-            std::cout << "Initialized distance constraint." << std::endl;
+
+            int verbosity_level_ = 0;
```

Comment:
`int verbosity_level_` is a property in the base Constraint class. This will make that property useless, so remove it. If verbosity_level_ (and in fact all integers) is not explicitly set, they are 0.